samples/notebooks/prediction/SDK_Custom_Predict.ipynb

```diff
@@ -893,7 +895,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Authorize gcloud to access the Cloud Platform with Google user credentials. This step requires to put user inputs. If you are using a non-interactive shells, you should open a terminal to run this command. If you are using Vertex Workbench, you will need to open [a terminal](https://jupyterlab.readthedocs.io/en/stable/user/terminal.html) to run this command."
+    "Authorize gcloud to access the Cloud Platform with Google user credentials. This step requires to put user inputs. If you are using a non-interactive shells, you should open a terminal to run this command. If you are using Vertex Workbench, you will need to open [a terminal](https://jupyterlab.readthedocs.io/en/stable/user/terminal.html) to run this command.\n",
```

Comment:
Please apply similar rewording here.

Reply:
Done.

Reply:
Actually, let's not repeat everything, and simply say...

Next, authorize gcloud to access the Cloud Platform with Google user credentials. Similar to the previous step, this also requires user inputs. Instructions for the previous step apply here as well.